pwa/src/apiService/apiService.ts

```diff
@@ -16,6 +16,7 @@ import ApiCalls from "./resources/apiCalls";
 import Subscriber from "./resources/subscriber";
 import Collection from "./resources/collection";
 import { isLoggedIn, logout, validateSession } from "../services/auth";
+import Repository from "./resources/repository";
```

Comment:
All resources are currently imported above the services, might be nice to move this import one line up, above the services.